### ISSUE TITLE
Windows install with python.org python

### ISSUE THREAD

**Issue by jtniehof** [spacepy/spacepy#155]
We build the Windows installer in an Anaconda environment but need to make sure the resulting installer will work in an install from python.org. @drsteve and I have been discussing this offline but want to capture it here.

Test: download latest Windows installer from python.org, install, run `python -m pip install numpy python-dateutil`, run SpacePy Windows installer, see if can import irbempy (since it's our difficult library.)

**Comment by drsteve** [comment]
I can check this after work, but I was making sure I grabbed the latest installer of the _same version_ of Python. So in my testing, where I discovered the libwinpthreads DLL was missing, I was using Python 3.6.8 in my conda install to build the installer, and 3.6.8 (from python.org) as my non-conda for testing.

To transfer notes from the other convo to this issue, the fact that the DLL was missing wasn't an issue in the conda environment because all of the MINGW runtime libraries are in the path. Once you come out of conda, that's no longer true. Copying the missing DLL from the conda MINGW stash to the non-conda install location meant I could import irbempy with no more errors.
